### DISCUSSION TITLE
Paperless NGX - Update Script Fehler

### DISCUSSION THREAD

**Discussion by redschumi** [tteck/Proxmox#3350] [Help Needed]
Hallo,

ich wollte heute meinen Paperless Container updaten. Die Installation wurde seinerzeit mit dem Script von
https://tteck.github.io/Proxmox/ durchgeführt. Gelegentlich Updates waren bisher immer ohne Fehler. Bis heute.
Update von 2.7.2 auf. 2.10.2 mittels “update” auf der Konsole. Danach ist die Paperless Instanz leider nicht mehr
konsistent. Web interface ist noch erreichbar, Dokumente werden nicht angezeigt. Folgende Fehler sind aufgetreten.

Script bricht ab:

“cat: /opt/Paperless-ngx_version.txt: No such file or directory
• Stopped all Paperless-ngx Services
\ Updating to v2.10.2 [ERROR] in line 83: exit code0: while executing command
/usr/bin/python3 manage.py migrate &> /dev/null”

Im Webinterface kann ich keine Dokumente anzeigen. Unter “Dateiaufgaben - Abgeschlossen” sowie im entsprechenden Ordner im Dateisystem sind
die Dokumente vorhanden: Fehler im Webinterface siehe Screenshot 2.
Proxmox Installation wurde vor Update Paperless upgedated. Reboot usw. ist erfolgt.
Log im Proxmox zeigt keine Fehler.
Die Version ist laut Webinterface auch upgedated auf 2.10.2.
Wenn ich das Updatescript erneut aufrufe, wird mir wieder das Update auf 2.10.2 angeboten, was nach Start des Script
immer wieder auf den gleichen Fehler läuft.

Habe jetzt erstmal das Backup zurückgespielt.

Evtl. hat wer eine Lösung oder einen Tipp für mich?
![IMG_1002](https://github.com/tteck/Proxmox/assets/6382306/bcabeb1d-041c-4016-9e60-a184fd78b83f)
![IMG_1001](https://github.com/tteck/Proxmox/assets/6382306/aff85ab8-a231-413c-8186-8573c0f47685)

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/2925

**Comment by suxlex** [comment]
I had the problem when updating paperless-ngx from version 2.7.2 to 2.8.6 - I then followed the tips and instructions in #2925 exactly and worked through them. After that I had version 2.8.6 (and the DB to UTF8)

Now I wanted to update from 2.10.2 to 2.11.0 and again I get this error as shown above.
I have tried all the tips described above again - but this time I can't get to 2.11.0.
What else could I try?

Linux-6.8.8-2-pve-x86_64-with-glibc2.36

**Reply by tteck** [reply]
English please
